### ISSUE TITLE
Plone 6.0.14 plone.app.discussion 4.1.4 cannot using Captcha Norobots?

### ISSUE THREAD

**Issue by mactrash** [collective/collective.z3cform.norobots#27]
The Captcha just not show

**Comment by 1letter** [comment]
@mactrash i test the addon in a plain coredev buildout. i can't confirm the issue. please check 
 your settings.

Please check that norobots is installed in the addon section

Please check the settings of discussion addon:

- enable discussion
- enable anonymous comments
- enable norobots captcha provider

logout from your site an go to a content with enabled comments, you should see the norobots widget.

![Screenshot from 2025-01-02 13-14-50](https://github.com/user-attachments/assets/92a5181c-533e-4b98-bff8-31dbfb429d75)

**Comment by mactrash** [comment]
Strange here is the test I just do now.
![Uploading Screenshot 2025-01-02 at 8.50.42 PM.png…]()

**Comment by 1letter** [comment]
@mactrash Please share your settings of plone.app.discussion and a screen of installed addons.

**Comment by 1letter** [comment]
@mactrash i have tested it with a plain installation of Plone 6.0.13 and an upgrade to Plone 6.0.14 but i can't reproduce the issue. Are you a anonymous user in your example above?

**Comment by mactrash** [comment]
@1letter 

Yes I build the 6.0.14 fresh and just active one addon and load the test news view in a new private browser window. Really strange.

I have a working 6.0.13 site, and just discover the Captcha not show after upgrade to 6.0.14. So I fallback to 6.0.13

**Comment by 1letter** [comment]
can you test the following steps please:

```
cd /tmp
```

```
python3 -m venv ./venv
```

```
source venv/bin/activate
```

```
touch requirements.txt && touch constraints.txt && touch instance.yaml
```

```
vim requirements.txt
```

add these lines to requirements.txt

```
-c constraints.txt
Plone
plone.app.upgrade
collective.z3cform.norobots
cookiecutter
```

add these lines to constraints.txt

```
-c https://dist.plone.org/release/6.0.14/constraints.txt
```

add these lines to instance.yaml

```
default_context:
  initial_user_name: "admin"
  initial_user_password: "admin"
  wsgi_listen: "localhost:8080"
  debug_mode: true
  verbose_security: true
  db_storage: "direct"
```

install all Plone dependencies

```
pip install -r requirements.txt
```

Install a testinstance

```
cookiecutter -f --no-input --config-file instance.yaml https://github.com/plone/cookiecutter-zope-instance
```

Start Plone in foreground mode

```
runwsgi -v instance/etc/zope.ini
```

- Go to http://localhost:8080 in your browser and add a Classic UI site
- activate collective.z3cform.norobots in http://localhost:8080/Plone/prefs_install_products_form
- add the discussion settings in the discussion controlpanel
- add a news item in the news folder and publish it
- and activate the discussion behavior in the settings tab of the news item
- logout
- go to your news item

**Comment by 1letter** [comment]
@mactrash can you test the manual above please?
